jest/defs.bzl

```diff
@@ -133,23 +155,49 @@ def jest_test(
     """
     tags = kwargs.pop("tags", [])
 
+    snapshot_data = []
     snapshot_files = []
+    snapshot_directories = []
     if snapshots == True:
-        snapshots = "__snapshots__"  # default jest snapshots directory
-    if type(snapshots) == "string":
-        snapshot_files = native.glob(["{}/**".format(snapshots)])
+        snapshot_directories = [to_label("__snapshots__")]  # default jest snapshots directory
+        snapshot_data = native.glob(["__snapshots__/**"])
+    elif type(snapshots) == "string":
+        snapshot_directories = [to_label(snapshots)]
+        snapshot_data = native.glob(["{}/**".format(snapshots)])
     elif type(snapshots) == "list":
-        snapshot_files = snapshots
-    elif snapshots != None:
-        fail("snapshots expected to be a boolean, string or list")
+        for snapshot in snapshots:
+            snapshot_label = to_label(snapshot)
+            if snapshot_label.package != native.package_name():
+                msg = "Expected jest_test '{target}' snapshots to be in test target package '{jest_test_package}' but got '{snapshot_label}' in package '{snapshot_package}'".format(
+                    jest_test_package = native.package_name(),
+                    snapshot_label = snapshot_label,
+                    snapshot_package = snapshot_label.package,
+                    target = to_label(name),
+                )
+                fail(msg)
+            if snapshot_label.name.endswith(snapshots_ext):
+                snapshot_files.append(snapshot_label)
+                snapshot_data.append(snapshot_label)
+            else:
+                snapshot_directories.append(snapshot_label)
+                snapshot_data.extend(native.glob(["{}/**".format(snapshot)]))
+
+    elif snapshots != False:
```

Comment:
Isn't changing `None` to `False` a breaking change?

Reply:
I changed the default to False

Reply:
but I suppose I could allow for both False & None here to mean the same thing

Reply:
Fixed

Reply:
If someone override that default with `None` this would make their build fail.

What was the reason for changing it?

Reply:
None is not as description as False in the docs IMO. But I changed it to accept both False & None so it won't be breaking.